### ISSUE TITLE
Option to ignore file tags

### ISSUE THREAD

**Issue by scottTomaszewski** [roovo/obsidian-card-board#208]
I write tasks in my daily notes which are built from a template that includes file tags in the frontmatter:

```
---
tags: daily_note, journal
---

- [ ] Do stuff #in-progress
- [ ] Do more stuff
```

As a result, every task will automatically get these tags applied which makes the "Untagged" column of a board unable to be used.

![image](https://github.com/roovo/obsidian-card-board/assets/5295276/655dbd88-177a-4490-b942-18368127343f)

In the above screenshot, I would expect there to be "Do more stuff" in the "Untagged" column, but its not there because of the file tags of "daily_note" and "journal".

If I remove the file tags, this does work, but I would prefer if I could keep the file tags and have the option to ignore them when applying filters
![image](https://github.com/roovo/obsidian-card-board/assets/5295276/0cf9e0ee-9264-4cfb-8f4a-024ab2c5d519)

Goal here is to add an option to the board settings that allows a user to toggle whether column filters ignore file tags

**Comment by roovo** [comment]
Would adding an 'Other Tags' column get you what you are after.  This should include tasks which do have tags other than the ones being used for columns on the board. ...although I am guessing that the issue here is that it will also include the 'Do stuff' card as this also has other tags on it!

I do plan to re-visit tag based columns at some point as there are some other useful cases which could sensibly supported so great to have this suggestion!  It will be a while tho as my current drag-and-drop work has quite a bit of work yet to be done.

Thank you :)

**Comment by scottTomaszewski** [comment]
> Would adding an 'Other Tags' column get you what you are after. This should include tasks which do have tags other than the ones being used for columns on the board. ...although I am guessing that the issue here is that it will also include the 'Do stuff' card as this also has other tags on it!

Yes, I tried this and the `#in-progress` tag was also included in the "Other Tags" column. 

> I do plan to re-visit tag based columns at some point as there are some other useful cases which could sensibly supported so great to have this suggestion! It will be a while tho as my current drag-and-drop work has quite a bit of work yet to be done.

No worries!  I cut the file tags from the frontmatter for now and it works great.  Had to adjust some dataview scripts, but thats not a big deal.  Looking forward to drag and drop! Thank you!
